packages/discord.js/src/structures/ThreadChannel.js

```diff
@@ -230,6 +230,15 @@ class ThreadChannel extends Channel {
     return this.guild.channels.resolve(this.parentId);
   }
 
+  /**
+   * The URL to the channel
+   * @type {string}
+   * @readonly
+   */
+  get url() {
+    return `https://discord.com/channels/${this.guild.id}/${this.parent.id}/${this.id}`;
```

Comment:
This is not correct, just remove it and keep the one from Channel

Reply:
Mind explaining why? The URL for threads is channels/guild_id/channel_id/thread_id.

Reply:
Simply because it isn’t. The format is the exact same one as the Channel one

Reply:
Ah, I see that without referencing the parent channel it still works. Is it still worth removing this implementation? I feel this one is more explicit and better.

Reply:
That URL links to the thread’s starter message and not the thread channel, so it’s not even correct

Reply:
It is wrong. This does not take you to the channel, but the message of the thread. If the thread was orphaned, this would go nowhere.